src/ucs/arch/aarch64/cpu.h

```diff
@@ -40,4 +40,9 @@ static inline ucs_cpu_model_t ucs_arch_get_cpu_model()
     return UCS_CPU_MODEL_UNKNOWN;
 }
 
+static inline int ucs_arch_get_cpu_flag()
```

Comment:
I would like to differ cases when we are able to detect and not.
UNKNOWN means that for some reason we can not detect.
0 means that we checked but cpu set is OFF